src/routes/CancelledPage.tsx

```diff
@@ -16,9 +16,8 @@ import {
 export default function CancelledPage() {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
-  const redirectUrl =
-    (getSessionItem(SessionItems.cart) as Cart | undefined)?.returnUrls
-      .returnCancelUrl || "/";
+  const cart = getSessionItem(SessionItems.cart) as Cart | undefined;
+  const redirectUrl = cart?.returnUrls.returnErrorUrl || "/";
```

Comment:
here we are in the user canceled page, why use error url here instead of the cancel one?
```suggestion
  const redirectUrl = cart?.returnUrls.returnCancelUrl || "/";
```

Reply:
Copy-paste miskate, take you